rpc/board.proto

```diff
@@ -51,3 +51,15 @@ message RequiredTool {
   string version = 2;
   string packager = 3;
 }
+
+message BoardAttachReq {
+  Instance instance = 1;
+  string boardURI = 2;
+  string sketchPath = 3;
+  string boardFlavour = 4;
+  string searchTimeout = 5;
+}
+
+message BoardAttachResp {
+  TaskProgress TaskProgress = 1;
```

Comment:
in proto files we do not use camel case but snake case:
`boardURI` -> `board_uri`
`sketchPath` -> `sketch_path`
`boardFlavour` -> `board_flavour`
`searchTimeout` -> `search_timeout`
`TaskProgress` -> `task_progress`

Reply:
ok, done